docs/quickstarts/ratify-with-venafi.md

```diff
@@ -0,0 +1,209 @@
+# Ratify with Venafi CodeSign Protect
+
+This guide will explain how to get started with Ratify and the [Venafi CodeSign Protect notation plugin](https://venafi.com/codesign-protect/). This will involve setting up necessary the necessary components, and configuring them properly. Once everything is set up we will walk through a simple scenario of verifying the signature on a container image at deployment time.
```

Comment:
```suggestion
This guide will explain how to get started with Ratify and the [Venafi CodeSign Protect notation plugin](https://venafi.com/codesign-protect/). This will involve setting up necessary components, and configuring them properly. Once everything is set up we will walk through a simple scenario of verifying the signature on a container image at deployment time.
```

Reply:
fixed.